generator/.DevConfigs/40980b2b-ec4c-4081-9a64-6a8a4a47d6bc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "services": [
+      {
+          "serviceName": "S3Control",
```

Comment:
How did the dry-run succeed if you changed `S3` but this says `S3Control`?

Reply:
I've no idea 😅, will update it and do another dry run.

Reply:
I'll follow up on this, the original DevConfig should not have succeeded.